runtime/compiler/CMakeLists.txt

```diff
@@ -247,10 +294,18 @@ create_omr_compiler_library(NAME j9jit
 	FILTER ${REMOVED_OMR_FILES}
 )
 
-add_dependencies(j9jit
-	omrgc_hookgen
-	j9jit_tracegen
-)
+if(${JITSERVER_SUPPORT})
+	add_dependencies(j9jit
+		omrgc_hookgen
+		j9jit_tracegen
+		j9jit_proto
+	)
+else()
+	add_dependencies(j9jit
+		omrgc_hookgen
+		j9jit_tracegen
+	)
+endif()
```

Comment:
I don't think you need to split the use of `add_dependencies`; you should be able to add a second (conditional) call for
```
add_dependencies(j9jit j9jit_proto)
```